src/Protocol/MojangAPI.cpp

```diff
@@ -883,15 +694,15 @@ void cMojangAPI::Update(void)
 	}
 	if (!PlayerNames.empty())
 	{
-		LOG("cMojangAPI: Updating name-to-uuid cache for %u names", static_cast<unsigned>(PlayerNames.size()));
+		LOG("cMojangAPI: Updating name-to-uuid cache for %u names", PlayerNames.size());
```

Comment:
Is this safe? I believe `%u` expects `uint32_t`, while `PlayerNames.size()` is `size_t`, which is `uint64_t` on 64-bit platforms.